src/utils/fileType/FileRecordTypeChecker.h

```diff
@@ -62,6 +62,7 @@ class FileRecordTypeChecker {
 	bool isGzipped() const { return _isGzipped; }
 
 	void setBam(); //call only if you're SURE the file is BAM!
+	void setIsGroupBy(bool val) { _isGroupBy = val; } // When using groupBy,
```

Comment:
Just curious, where is setIsGroupBy(true) used?

Reply:
Sorry, I just saw this. It looks like the change list above may not correctly show everything.  In ContextBase::openFiles, there is the line: 
		frm->setIsGroupBy(_program == GROUP_BY);
It's set right before the call to FileRecordMgr::open, because the FRM does the type checking when opening the file. This will tell it not to in the case of groupBy.
This is admittedly a hack and could have been more general, at least like an _isUnstructured or somesuch instead of isGroupBy, but right now it's the only tool using tabbed non-BED files likes that.